Description: Fixed trashed binary symbols in neutron_api_audit_map.conf.patch
Author: Thomas Goirand <zigo@debian.org>
Bug-Ubuntu: https://launchpad.net/bugs/1584715
Forwarded: no
Last-Update: 2016-05-23

--- python-pycadf-2.0.1.orig/etc/pycadf/neutron_api_audit_map.conf
+++ python-pycadf-2.0.1/etc/pycadf/neutron_api_audit_map.conf
@@ -3,7 +3,7 @@
 # should match the endpoint type defined in service catalog
 target_endpoint_type = None
 
-​[custom_actions]
+[custom_actions]
 add_router_interface = update/add
 remove_router_interface = update/remove
 
@@ -12,7 +12,7 @@ remove_router_interface = update/remove
 floatingips = ip
 healthmonitors = healthmonitor
 health_monitors = health_monitor
-lb = None​
+lb = None
 members = member
 metering-labels = label
 metering-label-rules = rule
